src/icons/file-icon/FileIcon.tsx

```diff
@@ -190,31 +187,41 @@ const EXTENSIONS: { [key: string]: { [key: string]: string } } = {
         'qt',
         'wmv',
     ]),
-    IconFileWordDocument: mirror(['docx', 'doc', 'docm']),
-    IconFileXbd: mirror(['xbd']),
-    IconFileXdw: mirror(['xdw']),
-    IconFileZip: mirror(['rar', 'tgz', 'zip']),
+    FileWord: mirror(['docx', 'doc', 'docm']),
+    FileXbd: mirror(['xbd']),
+    FileXdw: mirror(['xdw']),
+    FileZip: mirror(['rar', 'tgz', 'zip']),
 };
 
 const getFileIconComponent = (extension = '') => {
     const extensionComponentName = Object.keys(EXTENSIONS).filter(
         extensionComponent => !!EXTENSIONS[extensionComponent][extension.toLowerCase()],
     )[0];
-    return extensionComponentName || 'IconFileDefault';
+    return extensionComponentName || 'FileDefault';
 };
 
-interface FileIconProps {
+export interface FileIconProps {
     /** Dimension of the icon. */
     dimension?: number;
     /** Extension of file to display icon for. Defaults to generic icon */
     extension?: string;
-    /** A text-only string describing the icon if it's not purely decorative for accessibility */
-    title?: string | React.ReactElement<string>;
 }
 
-const FileIcon = ({ dimension = 32, extension = '', title }: FileIconProps) => {
```

Comment:
removing `title` prop could be a breaking change for internal apps. for example, admin console uses this component so removing `title` prop can break tests that query by label